diff --git a/opt/datadog-agent/agent/checks.d/kubernetes.py b/opt/datadog-agent/agent/checks.d/kubernetes.py
index b9d899a..a726b3a 100644
--- a/opt/datadog-agent/agent/checks.d/kubernetes.py
+++ b/opt/datadog-agent/agent/checks.d/kubernetes.py
@@ -241,6 +241,16 @@ def _update_container_metrics(self, instance, subcontainer, kube_labels):
         if container_image:
             tags.append('container_image:%s' % container_image)
 
+            split = container_image.split(":")
+            if len(split) > 2:
+                # if the repo is in the image name and has the form 'docker.clearbit:5000'
+                # the split will be like [repo_url, repo_port/image_name, image_tag]. Let's avoid that
+                split = [':'.join(split[:-1]), split[-1]]
+
+            tags.append('image_name:%s' % split[0])
+            if len(split) == 2:
+                tags.append('image_tag:%s' % split[1])
+
         try:
             cont_labels = subcontainer['spec']['labels']
         except KeyError:
